data/brands/amenity/bank.json

```diff
@@ -21,6 +21,16 @@
     }
   },
   "items": [
+     {
+      "displayName": "German American Bank",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "bank",
+        "brand": "German American Bank",
```

Comment:
```suggestion
        "brand": "German American",
```
Based on their branding, I would just call it "German American" in both the `brand` and `displayName` props.

<img width="300" alt="Screenshot 2023-07-21 at 12 30 07 PM" src="https://github.com/osmlab/name-suggestion-index/assets/38784/5ec740c0-da41-440f-afa9-6db478db9c1b">